src/game/client/render_map.cpp

```diff
@@ -454,3 +455,151 @@ void CRenderTools::RenderTilemap(const CTile *pTiles, int w, int h, float Scale,
 
 	Graphics()->QuadsEnd();
 }
+void CRenderTools::RenderWaterMap(const CTile* pTiles, int w, int h, float Scale, vec4 Color, int RenderFlags,
```

Comment:
@Dune-jr @heinrich5991 is it possible to render images with a bigger Anti-Aliasing? Currently you can see the stair-artifacts of the individual sprites. It's not really bad, but I think it maybe could aproved on with that feature